includes/Traits/FieldableTrait.php

```
@@ -438,106 +452,761 @@ public function update_post_meta( $meta_vars, $post_id ) {
+            if ( isset( $taxonomy['woo_attr'] ) && wpuf_is_checkbox_or_toggle_on( $taxonomy['woo_attr'] ) ) {
+                $woo_attr[$taxonomy_name] = $this->build_woo_attribute( $taxonomy_name, $taxonomy );
+            }
+        }
+    }
 
-                    if ( $taxonomy['type'] == 'text' ) {
-                        wp_set_object_terms( $post_id, $taxonomy_name, $taxonomy['name'] );
+    /**
+     * Handle product type taxonomy
+     */
+    protected function handle_product_type( $post_id, $taxonomy_name, $posted_terms ) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$taxonomy_name'. (Unused Code Rules)

(UnusedFormalParameter)